src/main/java/racingcar/UserView.java

```diff
@@ -0,0 +1,54 @@
+package racingcar;
+
+import java.util.ArrayList;
+
+import static camp.nextstep.edu.missionutils.Console.readLine;
+
+public class UserView {
+    public static String readCarName(){
+        printCarNameGuide();
+        return readLine();
+    }
+
+    private static void printCarNameGuide(){
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+    }
+
+    public static String readAttemptNum(){
+        printAttemptNumGuide();
+        return readLine();
+    }
+
+    private static void printAttemptNumGuide(){
+        System.out.println("시도할 횟수는 몇 회인가요?");
+    }
+
+    public static void printResultGuide(){
+        System.out.println("실행 결과");
+    }
+
+    public static void printPlayerAttemptResult(String name, int count){
+        StringBuilder playertResult = new StringBuilder();
+        playertResult.append(name);
+        playertResult.append(" : ");
+        playertResult.append("-".repeat(count));
+        System.out.println(playertResult);
+    }
+
+    public static void printWinner(ArrayList<String> winners){
+        System.out.print("최종 우승자 : ");
+        boolean coChampions = false;
+        for (String winner: winners){
+            if (coChampions){
+                System.out.print(", ");
+            }
+            else coChampions = true;
+
+            System.out.print(winner);
```

Comment:
String.join()을 이용하면 단독 우승자, 공동 우승자를 따로 고려하지 않으면서도 간단하게 winners의 요소들을 ", "로 연결해 출력할 수 있습니다!

```suggestion
        System.out.print("최종 우승자 : " + String.join(", ", winners));
```

Reply:
오 꿀팁 감사합니다.